docs/build_docs.py

```diff
@@ -0,0 +1,296 @@
+import logging
+import argparse
+import logging.handlers
+import shutil
+import os
+
+import rich
+from rich.logging import RichHandler
+from rich.traceback import install as install_rich_tracebacks
+import rich_argparse
+
+
+def init_logger(use_rich):
+    logger = logging.getLogger("build_docs")
+    if use_rich:
+        install_rich_tracebacks()
+        logging_handlers = [RichHandler(rich_tracebacks=True)]
+    else:
+        logging_handlers = [logging.StreamHandler()]
+
+    logging.basicConfig(level=logging.DEBUG, datefmt="[%X]", handlers=logging_handlers)
+    logger.debug("Program initialized")
+    return logger
+
+
+def parse_args():
```

Comment:
Maybe name this something different since `parse_args` is the name of a function from argparse.

Reply:
Fixed!